src/compiler/types.ts

```diff
@@ -3074,6 +3074,7 @@ namespace ts {
     export interface Type {
         flags: TypeFlags;                // Flags
         /* @internal */ id: number;      // Unique ID
+        /* @internal */ checker: TypeChecker;

```

Comment:
This should definitely be documented as only being present in debug mode... better, don't define it here at all and require a cast to `DebugType`.